src/stryker-cli.ts

```diff
@@ -25,40 +28,40 @@ export function run(log = console.log): Promise<void> {
 }
 
 function runLocalStryker() {
-  const strykerCli = localStrykerCli();
-  RequireWrapper.require(strykerCli);
+  const stryker = localStryker();
```

Comment:
Good naming ;)

Reply:
Yeah, `strykerCli` sounded strange to me. It is the cli... of stryker. But as this project is also called 'stryker-cli'...